R/Functions_L0_auxilary.R

```diff
@@ -1063,7 +1063,7 @@ getQueenYearOfBirth <- function(x, simParamBee = NULL) {
     }
   } else if (isColony(x)) {
     if(packageVersion("AlphaSimR") > package_version("1.5.3")){
```

Comment:
@janaobsteter Best to just fix package version in DESCRIPTION and then we don't need this code duplication.